src/main/scala/Serdes.scala

```diff
@@ -555,7 +556,8 @@ class TLDesser(w: Int, params: Seq[TLClientParameters], hasCorruptDenied: Boolea
   val node = TLClientNode(params.map(client =>
       TLMasterPortParameters.v1(Seq(client))))
 
-  lazy val module = new LazyModuleImp(this) {
+  lazy val module = new Impl
+  class Impl extends LazyModuleImp(this) {
```

Comment:
Why is this needed/done?

Reply:
Result of change in scala2.13. This change was applied via scalafix, and all rocket-chip dependencies on 2.13 had this change applied